src/vs/workbench/browser/parts/editor/editorTabsControl.ts

```diff
@@ -202,6 +234,10 @@ export abstract class EditorTabsControl extends Themable implements IEditorTabsC
 	}
 
 	protected updateEditorActionsToolbar(): void {
+		if (!this.editorActionsEnabled) {
```

Comment:
There is another method below `protected clearEditorActionsToolbar(): void {` which I wonder how we should deal with it based on the setting. What does it mean to "clear" the editor actions toolbar, when it is rooted in the title?

Reply:
If clear is called and the actions are in the title bar then `this.editorActionsToolbar` will be undefined and nothing will and should happen. I'll write it a bit more explicit.

`clearEditorActionsToolbar()` is the same as `updateEditorActionsToolbar()` just a bit faster when knowing there are no editor actions.